integrations/QuoraPixel/browser.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable */
+import logger from "../../utils/logUtil";
+import { NAME } from "./constants";
+import { getHashFromArray } from "../utils/commonUtils";
+
+class QuoraPixel {
+  constructor(config) {
+    this.name = NAME;
+    this.pixelId = config.pixelId;
+    this.eventsToQPEvents = config.eventsToQPEvents;
+  }
+
+  loadScript() {
```

Comment:
we have a scriptloader function in the repo. Can we tally these two and see if that can work in this place?

Reply:
@Gauravudia did you check this?

Reply:
this is not working